tests/parsers/test_mkdocs.py

```diff
@@ -0,0 +1,51 @@
+from visivo.parsers.mkdocs import Mkdocs
+import yaml
+
+TEST_MKDOCS_OBJECT = {
+        'nav': [
+            {'getting started': 'index.md'},
+            {'including': 'index.md'},
+            {'reference': [
+                {'Configuration': {'trace': [], 'chart': []}},
+                {'cli': 'index.md'}
+            ]}
+        ],
+        'extra_css': [{'Configuration':'something'}]
+    }
+
+mkdocs = Mkdocs()
+
+def test_nav_configuration():
+    nav_config = mkdocs.get_nav_configuration()
+    assert len(nav_config) > 5
+
+def test_docs_content_from_pydantic_model():
+    trace_md = mkdocs.get_md_content('Trace')
```

Comment:
Is there something magical about 50?

Reply:
Nothing magical, just wanted to check that the string isn't empty and has some content.